src/cmds/versions/create.js

```diff
@@ -67,19 +66,45 @@ module.exports = class CreateVersionCommand {
       );
     }
 
-    if (!fork) {
+    let versionList;
+    if (!opts.fork) {
       versionList = await fetch(`${config.get('host')}/api/v1/version`, {
         method: 'get',
         headers: cleanHeaders(key),
       }).then(res => handleRes(res));
     }
 
-    const versionPrompt = promptOpts.createVersionPrompt(versionList || [{}], {
-      newVersion: version,
-      ...opts,
-    });
+    const questions = [
+      {
+        type: 'select',
+        name: 'fork',
+        message: 'Which version would you like to fork from?',
+        skip: () => opts.fork,
+        choices: (versionList || [{}]).map(v => ({ message: v.version, value: v.version })),
+      },
+      {
+        type: 'confirm',
+        name: 'main',
+        message: 'Would you like to make this version the main version for this project?',
+        skip: () => 'main' in opts,
+      },
+      {
+        type: 'confirm',
+        name: 'beta',
+        message: 'Should this version be in beta?',
+        skip: () => 'beta' in opts,
```

Comment:
So just to confirm (since I'm still getting familiar with `enquirer`)... if someone omits the `--beta` flag in a CI environment, `opts.beta` will be populated but set to `false`?

Reply:
Ignore me, I asked that before I saw this chunk: https://github.com/readmeio/rdme/blob/f16ae276b5101ac5174f7bc932c94da4e2797060/src/lib/enquirer.js#L22-L30